### ISSUE TITLE
"pinhead" wave display

### ISSUE THREAD

**Issue by ossilator** [sonic-visualiser/sonic-visualiser#71]
when a wave is zoomed in so much that individual samples become visible, they are interpolated. this can be turned off, which makes the points just "hang in the air". to make the waves appear more coherent, it makes sense to draw a perpendicular line to the y axis from each point, thus making the wave appear as a "row of pins".

on a somewhat related note, the points could/should become bigger once the spacing between them increases sufficiently.

i'm using the wave display from audacity as a reference.
(on a totally unrelated note, i'd appreciate being able to change the modifier mapping for the mouse wheel to match audacity, because finger memory. anyway ...)

**Comment by cannam** [comment]
There was some brief discussion about this in #34 - a reason for omitting the stems is to avoid interfering with anything that is displayed behind the waveform.

(Which mouse modifiers are you referring to in Audacity? I don't use it very often and tend to think of it as quite hard to navigate in, perhaps because it doesn't use the same controls as SV...)

**Comment by ossilator** [comment]
yeah, i don't think enabling this unconditionally would be universally appreciated. a separate checkbox (which is best for backwards compat) or a tri-state dropdown (which is best for usability) would do.

(in the wave graph area, ctrl zooms horizontally and shift moves horizontally, while in the wave scale area, the modifiers have the same effect, but vertically. without modifiers, the window content is scrolled as usual, which is very helpful, as you can resize the waves vertically as needed, thus potentially exceeding the screen space).
